variants/heltec_wireless_stick_lite_v3/pins_arduino.h

```diff
@@ -0,0 +1,60 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define Wireless_Stick_Lite true
+#define DISPLAY_HEIGHT 0
+#define DISPLAY_WIDTH  0
+
+#define EXTERNAL_NUM_INTERRUPTS 16
+#define NUM_DIGITAL_PINS        40
+#define NUM_ANALOG_INPUTS       15
+
+#define analogInputToDigitalPin(p)  (((p)<20)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<40)?(p):-1)
+#define digitalPinHasPWM(p)         (p < 34)
+
+static const uint8_t LED_BUILTIN = 35;
+#define BUILTIN_LED  LED_BUILTIN // backward compatibility
+#define LED_BUILTIN LED_BUILTIN
+
+static const uint8_t TX = 44;
+static const uint8_t RX = 43;
+
+static const uint8_t SDA = 2;
+static const uint8_t SCL = 3;
+
+static const uint8_t SS    = 34;
+static const uint8_t MOSI  = 35;
+static const uint8_t SCK   = 36;
+static const uint8_t MISO  = 37;
+
+static const uint8_t A0 = 1;
+static const uint8_t A1 = 2;
+static const uint8_t A2 = 3;
+static const uint8_t A3 = 4;
+static const uint8_t A4 = 5;
+static const uint8_t A5 = 6;
+static const uint8_t A6 = 7;
+static const uint8_t A10 = 12;
+static const uint8_t A12 = 14;
+static const uint8_t A13 = 15;
+static const uint8_t A14 = 16;
+static const uint8_t A15 = 17;
+static const uint8_t A16 = 18;
+static const uint8_t A17 = 19;
+static const uint8_t A18 = 20;
```

Comment:
Should be analog ADC `A1` to `A19` for the ESP32-S3. See page 19:
https://www.espressif.com/sites/default/files/documentation/esp32-s3_datasheet_en.pdf

See also the Heltec Wifi Lora V3 pins:
https://github.com/espressif/arduino-esp32/blob/master/variants/heltec_wifi_lora_32_V3/pins_arduino.h

Reply:
Same comment as above about which pins to include. Either way works.

However, why do your GPIO pins change from being A+1 to A+2 starting at A10? That doesn't seem right?

Reply:
Fixed, it didn't make sense (see above)